data/weapons.txt

```diff
@@ -457,6 +457,32 @@ effect "proton impact"
 	"velocity scale" -.005
 
 
+outfit "Proton Turret"
+	category "Turrets"
+	cost 260000
+	thumbnail "outfit/proton turret"
+	"mass" 36
+	"outfit space" -36
+	"weapon capacity" -36
+	"turret mounts" -1
+	weapon
+		sprite "projectile/proton"
+		"hardpoint sprite" "hardpoint/proton turret"
+		"hardpoint offset" 14.
+		sound "proton"
+		"hit effect" "proton impact" 3
+		"inaccuracy" 4
+		"turret turn" 2.5
```

Comment:
Is this turn rate considered to be pre- or post- #3271 ?

Reply:
Post.